vhdbuilder/packer/install-dependencies.sh

```diff
@@ -199,23 +199,31 @@ installBpftrace
 echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
 
 if [[ $OS == $UBUNTU_OS_NAME && $(isARM64) != 1 ]]; then  # no ARM64 SKU with GPU now
-addNvidiaAptRepo
-installNvidiaDocker "${NVIDIA_DOCKER_VERSION}"
-downloadGPUDrivers
-retrycmd_if_failure 30 5 3600 wget "https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/fabricmanager-linux-x86_64-${GPU_DV}.tar.gz" || exit $ERR_GPU_DOWNLOAD_TIMEOUT
-tar -xvzf fabricmanager-linux-x86_64-${GPU_DV}.tar.gz -C /opt/azure
-mv /opt/azure/fabricmanager /opt/azure/fabricmanager-${GPU_DV}
-echo "  - nvidia-docker2 nvidia-container-runtime" >> ${VHD_LOGS_FILEPATH}
-downloadNvidiaContainerRuntime || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
-{
-  echo "  - nvidia-container-runtime=${NVIDIA_CONTAINER_RUNTIME_VERSION}";
-  echo "  - nvidia-gpu-driver-version=${GPU_DV}";
-  echo "  - nvidia-fabricmanager=${GPU_DV}";
-} >> ${VHD_LOGS_FILEPATH}
-if grep -q "fullgpu" <<< "$FEATURE_FLAGS"; then
+  # for dedicated GPU VHD, install older driver version compatible with all VM sizes
+  # consider this informal "deprecation" since we can't support one version right now.
+  if grep -q "fullgpu" <<< "$FEATURE_FLAGS"; then
     echo "  - ensureGPUDrivers" >> ${VHD_LOGS_FILEPATH}
+    export GPU_DV=470.57.02
```

Comment:
this forces the same version we have been using on the gpu vhd to avoid breaking those users. they won't get v5 support for now. a subtle nudge to get them on the normal vhd since everything is precached with no egress there anyway..